sdk/cdn/azure-mgmt-cdn/azure/mgmt/cdn/_configuration.py

```diff
@@ -12,14 +12,13 @@
 from azure.core.pipeline import policies
 from azure.mgmt.core.policies import ARMHttpLoggingPolicy
 
-from ._version import VERSION
-
 if TYPE_CHECKING:
     # pylint: disable=unused-import,ungrouped-imports
     from typing import Any
 
     from azure.core.credentials import TokenCredential
 
+VERSION = "unknown"
```

Comment:
It seems `package_vesion` is missing from readme.python.md, please help fix it and regenerate SDK.